src/applications/vaos/appointment-list/components/AppointmentsPageV2/index.jsx

```diff
@@ -135,6 +136,9 @@ export default function AppointmentsPageV2() {
       if (featureStatusImprovement) {
         document.title = `${pageTitle} | VA online scheduling | Veterans Affairs`;
         scrollAndFocus('h1');
+        recordEvent({
+          event: `${GA_PREFIX}-new-appointment-list`,
+        });
```

Comment:
Not sure this is the right place. I was expecting the event to be recorded when featureAppointmentList is true. Maybe in UpcomingAppointmentList.jsx

Reply:
@vbahinwillit Made that change, waiting for CI to complete.